packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -303,16 +310,14 @@ async function fetchLatestSnapshotCore(
                     sequenceNumber,
                     ops: snapshot.ops?.length ?? 0,
                     headers: Object.keys(response.requestHeaders).length !== 0 ? true : undefined,
-                    redirecttime: redirectTime,
-                    dnsLookuptime: dnstime,
-                    responsenetworkTime: responseTime,
-                    tcphandshakeTime: tcpHandshakeTime,
-                    secureconnectiontime: secureConntime,
-                    fetchstarttorespendtime: fetchStToRespEndTime,
-                    reqstarttorespendtime: reqStToRespEndTime,
-                    overalltime: overallTime,
-                    networktime: networkTime,
-                    clienttime: clientTime,
+                    redirectTime: redirectTime,
```

Comment:
Like you comment above with description of what browser provides, but it would be great to add similar comment that says what each telemetry property means. Such that developers looking at telemetry can quickly reason about it, without need to parse code above and understand transformation we do.